stack.yaml

```diff
@@ -87,177 +87,177 @@ nix:
 # package cardano-crypto
 #   tests: False
 #   benchmarks: False
-# 
+#
```

Comment:
I think that stack2cabal relies on these whitespaces and they shouldn't be trimmed. 

@paweljakubas, is your editor correctly picking up the config from: https://github.com/input-output-hk/cardano-wallet/blob/master/.editorconfig#L11-L12

Reply:
it seems so: 
```
# EditorConfig for /home/pawel/Work/IOHK/cardano-wallet/stack.yaml                                                                                                                                                 
indent_style = space                                                                                                                                                                                               
indent_size = 2                                                                                                                                                                                                    
end_of_line = lf                                                                                                                                                                                                   
charset = utf-8                                                                                                                                                                                                    
trim_trailing_whitespace = false                                                                                                                                                                                   
insert_final_newline = true                                                                                                                                                                                        
tab_width = 2
```

Reply:
I think `trim_trailling_whitespace=true` is a nice thing to have.

When auto-generated whitespace gets removed in a file that you edited, use your editor's git function to discard those irrelevant hunks.